data/rajmaurya-dev.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Raj Maurya",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer | Open source contributor | Building Projects",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/rajmaurya_dev",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://www.github.com/rajmaurya-dev/",
```

Not sure the `www.` is needed here.
```suggestion
      "url": "https://github.com/rajmaurya-dev/",
```